Gemfile

```diff
@@ -34,6 +34,8 @@ gem "redis", "~> 4.0"
 
```

Comment:
**[OPTIONAL]**

- [ ] Your pull request summary is not descriptive enough :fearful: Kindly add more details into your pull request summary to make your pull request professional :blush: